### ISSUE TITLE
z-index and html positioning causes tooltip to show behind bootstrap modal

### ISSUE THREAD

**Issue by crobinson42** [bsidelinger912/react-tooltip-lite#61]
When using a tooltip in a component that's inside a bootstrap4 modal, the tooltip cannot be seen.

A quick look at the DOM reveals the tooltip is being appended inside the `<body>` which is the primary reason for not showing in the modal situation.

**Recommendation** (let's discuss) to refactor the tooltip to append immediately after it's children/child component. I understand there might be some disturbance in very few situations where the DOM flow might render a space, but we should investigate this in a refactor effort.

@bsidelinger912 thoughts?

**Comment by bsidelinger912** [comment]
Actually rendering along side the component actually can cause many more issues with layout and z-index for other use cases, and would likely result in major regressions.  However, we def want this to work inside a bootstrap modal.  What would really help is a repro in a codepen (or similar shareable repro). I can set that up when I have time, but if you'd like to in the meantime, that'll allow me to get to it faster.

**Comment by crobinson42** [comment]
Ok - the solution for me when using bootstrap 4 is to use the same z-index as the bootstrap framework: https://github.com/twbs/bootstrap/blob/master/scss/_variables.scss#L708

z-index on the classes for `react-tooltip-lite` need to be set to 1070.

**Comment by crobinson42** [comment]
```
@import '~bootstrap/scss/_variables';

// temporary fix waiting on - https://github.com/bsidelinger912/react-tooltip-lite/issues/61
.react-tooltip-lite, .react-tooltip-lite-arrow {
  z-index: $zindex-tooltip !important;
}
```

**Comment by bsidelinger912** [comment]
@crobinson42 I think this is addressed with the new zIndex prop from the 1.11 release
